scripts/build/appveyor/configure_appveyor_VS2017.bat

```diff
@@ -13,6 +13,8 @@ cmake .. -G "Visual Studio 15 2017 Win64"
 -DMESH_MOVING_APPLICATION=ON                                                    ^
 -DSOLID_MECHANICS_APPLICATION=ON                                                ^
 -DCONSTITUTIVE_MODELS_APPLICATION=ON                                            ^
+-DADD_DEPENDENT_APPLICATIONS=ON                                                 ^
```

Comment:
@josep-m-carbonell  I like the concept of ADD_DEPENDENT_APPLICATIONS. However, in the case of this specific file, it is good to know which apps are being compiled. I've checked this file many times, and I know that @roigcarlo did as well. And ADD_DEPENDENT_APPLICATIONS is something that hides the total number of applications being compiled. IMO it is cool for the regular use in our own configures, but too opaque here...

Reply:
Ok, its better to show the applications here. I will commit the correction.